concepts/05 UI Components/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,270 @@
+To submit a form, add a [Button Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/ButtonItem/) and set its [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property to **true**. The Form can be submitted to a server only if input validation is successful. 
+
+The **useSubmitBehavior** property requires that you wrap the dxForm in the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" target="_blank">form</a> element. You should also set the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault" target="_blank">preventDefault</a> property to **true** to override the HTML form submit event as shown in the code example.
+
+The code below shows how to add a submit button, but does not show how to implement the backend. The example displays a fake confirmation message after a timeout:
```

Comment:
"a fake confirmation message" Why do you refer to it as **fake**?

Reply:
In real applications, a confirmation message (or an error message) is received from a server and depends on various conditions. It also takes some time to receive the server's response. The sample application described in this section just displays predefined message in one second after the click on the 'Submit' button regardless of any conditions.

Reply:
I would mot use 'fake' in any context. 

```suggestion
The code below shows how to add a submit button, but does not show how to implement the backend. The example displays a confirmation message after a timeout:
```